model/task/task.go

```diff
@@ -1039,6 +1039,7 @@ func SetTasksScheduledTime(tasks []Task, scheduledTime time.Time) error {
 // operation will select tasks from all distros.
 func UnscheduleStaleUnderwaterTasks(distroID string) (int, error) {
 	query := scheduleableTasksQuery()
+	hint := "distro_1_status_1_activated_1_priority_1"
```

Comment:
What index is it using instead of this one?

Is it worth commenting why the explicit hint is necessary instead of just letting the query planner pick?

Reply:
its currently using `status_1_depends_on.status_1_depends_on.unattainable_1` if we dont specify 

if you think a comment might be useful i could add that in though!

Reply:
I think saying why we have to tell the query planner to not plan on our behalf makes sense. Comments are valuable when explaining why we have to do something unexpected, especially special cases like overriding the built-in query planner's decision, which we rarely have to do.

Reply:
Can you build this like this instead of hardcoding? https://github.com/evergreen-ci/evergreen/blob/8e010917fc3774e4f32d8c2843c6d23cbed623a5/model/host/host.go#L1696
I agree with commenting why this is needed.